tests/test_nn/test_representation.py

```diff
@@ -184,7 +187,7 @@ def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMa
         return kwargs
 
 
-@unittest.skipIf(error is not None, "Need to install `torch_geometric`")
+@unittest.skipIf(torch_geometric is None, "Need to install `torch_geometric`")
```

Comment:
you can store the decorator as a constant and re-use it to avoid code duplication

Reply:
f344154